README.md

```diff
@@ -127,16 +127,18 @@ global.instancename = dev-from-ide
 
 # DE4A Kafka settings
 de4a.kafka.enabled=false
+# Enables Kafka connection via HTTP (it must be used when HTTP connections are being established via proxy)
+de4a.kafka.http.enabled=false
 # Kafka server address (default: de4a-dev-kafka.egovlab.eu:9092)
 de4a.kafka.url=de4a-dev-kafka.egovlab.eu:9092
-# Add any suffix if you have multiple Connector instances on the same kafka server (default: DE4A-CONNECTOR)
-de4a.kafka.topic=DE4A-CONNECTOR
+# Add any suffix if you have multiple Connector instances on the same kafka server (default: de4a-connector)
+de4a.kafka.topic=de4a-connector
 
 # toop legacy kafka properties (Don´t touch)
 toop.tracker.enabled = ${de4a.kafka.enabled}
```

Comment:
For this to work we would need to point the toop tracker to the de4a kafka instance. Also, the toop tracker will NOT work if http mode is enabled.

Reply:
Most likely the toop tracker will crash since it will not accept the url with https in it

Reply:
Indeed it should be disabled in any case, at this stage, it makes no sens send any toop logs to kafka